### PR TITLE
 GEOMESA-2169 Create a FSDS Tutorial

### PR DESCRIPTION
**Pull request by aheyne**
Depends on #58 and GeoMesa #1862

### CODE REVIEW

**Comment by aheyne** [comment]
Build is failing because package doesn't exist 
> /home/travis/build/geomesa/geomesa-tutorials/geomesa-quickstart-fsds/src/main/java/org/geomesa/example/fsds/FSDSQuickStart.java:[24,51] package org.locationtech.geomesa.fs.storage.interop does not exist

**Comment by elahrvivaz** [comment]
@aheyne can you rebase this on master?

**Comment by elahrvivaz** [comment]
and just double check that it builds now that I've merged the geomesa interop stuff

File: geomesa-quickstart-fsds/src/main/java/org/geomesa/example/fsds/FSDSQuickStart.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    public FSDSQuickStart(Map<String, String> params) { super(params); }
+
+    public FSDSQuickStart(String[] args) throws ParseException {
+        super(args, new FileSystemDataStoreFactory().getParametersInfo());
+    }
+
+    @Override
+    public SimpleFeatureType createSchema(DataStore datastore, QuickStartData data) throws IOException {
+        SimpleFeatureType sft = data.getSimpleFeatureType();
```
Comment (elahrvivaz) [inline geomesa-quickstart-fsds/src/main/java/org/geomesa/example/fsds/FSDSQuickStart.java]: maybe we should break out `getSimpleFeatureType(QuickStartData data)` into a separate method we can override

File: geomesa-quickstart-fsds/src/main/java/org/geomesa/example/fsds/FSDSQuickStart.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        PartitionScheme scheme = CommonSchemeLoader.build("daily,z2-2bit", sft);
```
Comment (elahrvivaz) [inline geomesa-quickstart-fsds/src/main/java/org/geomesa/example/fsds/FSDSQuickStart.java]: can you add a comment here explaining what's going? makes the tutorial more accessible

File: geomesa-quickstart-fsds/src/main/java/org/geomesa/example/fsds/FSDSQuickStart.java (hunk 1 above)
Comment (elahrvivaz) [reply geomesa-quickstart-fsds/src/main/java/org/geomesa/example/fsds/FSDSQuickStart.java]: actually don't worry about it for now - I'll go ahead and merge this once you rebase, then update it in the new PR, which will also require a few tweaks here

**Pull request merged**